pkg/apis/options/legacy_options.go

```diff
@@ -48,14 +49,14 @@ func NewLegacyOptions() *LegacyOptions {
 
 		LegacyProvider: LegacyProvider{
 			ProviderType:          "google",
-			AzureTenant:           "common",
+			AzureTenant:           "",
```

Comment:
Can remove this line entirely I think

Reply:
I am nor sure anymore whether my assessment on sensible Azure defaults was correct.

Yes, "common" is the most permissive setting, _but_ all the other options for IdPs are also set to the most permissive.

Reply:
Still think we can remove this. The default is common and we can default that as part of the provider initialisation, that seems to be the most sensible place to have the default logic for that particular option